client/domain/CorrectnessTestObjectFactorySpec.js

```diff
@@ -48,4 +61,16 @@ describe('CorrectnessTestObjectFactory', function() {
       expect(['a', 'b']).toContain(correctnessTest.getAnyAllowedOutput());
     });
   });
+
+  describe('getMessage', function() {
+    it('should correctly get the message', function() {
+      var correctnessTest = CorrectnessTestObjectFactory.create({
+        input: 'cat',
+        allowedOutputs: ['a', 'b'],
+        message: "hello"
+      });
+
+      expect(correctnessTest.getMessage()).toMatch("hello");
```

Comment:
.toMatch(correctnessTest['message']), just in case someone changes it down the line.